apps/console/src/features/roles/components/role-list.tsx

```diff
@@ -336,6 +353,19 @@ export const RoleList: React.FunctionComponent<RoleListProps> = (props: RoleList
                     </ConfirmationModal>
                 )
             }
+            {
+                showDeleteErrorConnectedAppsModal && (
+                    <RoleDeleteErrorConfirmation
```

Comment:
If we are going to show a confirmation modal here as well for role deletion, I think delete button should not be shown at all in the role list item.

Reply:
We can't just hide the delete button. What can do is disable the button with a hint to the user.
What we have done in other places in the console is showing a Modal like this. Ex: Connections page